### ISSUE TITLE
Some suggestions for nested slides

### ISSUE THREAD

**Issue by Joliamendi** [elementor/elementor#19837]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

I am very excited to release nested element in the near future. So, I was going to request some important features to develop this widget. I hope it will be updated with the new version. Actually, my request is for full control over the arrows and Pagination of the slides widget.

### Describe the solution you'd like

1- Custom icon for arrows
2- Background color, padding, box shadow, border and border radius for arrows
3- Responsiveness of the image in the slides widget
4- Pagination can be displayed vertically
5- Custom position for arrows (x y)
6- border Radius for Slides image
7- center mode :

![download (1)](https://user-images.githubusercontent.com/103376243/190893623-b00d8ea4-2b1d-42b9-923a-b03f547cbe7c.png)

8- Pagination mode display option : 

![download (2)](https://user-images.githubusercontent.com/103376243/190893662-dc82a29a-f1d2-4175-bbdb-189b9f5ac914.png)

**Comment by nicholaszein** [comment]
Hello @Joliamendi!

Thanks for submitting your suggestions. Part of your request has been already implemented in the normal slider like custom icons for arrows, and border radius for slides. And other parts will be covered by the elements inside each slide (background color, padding, box shadow, etc.).

But we'll consider the other suggestions.

**Comment by SolunaCG** [comment]
I just found this thread and I wonder if "Nested Sliders" are still a thing cause we now have the "Nested Carousel" which is different though. If they were both combined into the "Nested Carousel" now then I´d love to see see things mentioned above, too! Plus what I´d like to add is a few more transitions just like we already have them in the old "Slides" widget in Elementor. "Slide" and "Fade" would already be great to have imo.

**Comment by 79ho3ein** [comment]
> I just found this thread and I wonder if "Nested Sliders" are still a thing cause we now have the "Nested Carousel" which is different though. If they were both combined into the "Nested Carousel" now then I´d love to see see things mentioned above, too! Plus what I´d like to add is a few more transitions just like we already have them in the old "Slides" widget in Elementor. "Slide" and "Fade" would already be great to have imo.

Exactly, it's strange that there is no fade effect for the new slider.
